src/components/Main.js

```diff
@@ -41,8 +47,13 @@ class Main extends React.Component{
   render (){
     
     return <div>
-      <span>Todo</span><input type="textbox" id="textbox1" />
-      <button onClick={this.addTodo}  >Add</button>
+     <form action="">
```

Comment:
why form tag is used and why action is empty string?

Reply:
Noted <form> as unnecessary.Will be removed